packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * @import {Zone} from '@agoric/zone';
+ * @import {TypedPattern} from '@agoric/internal'
+ * @import {StorageNode} from '@agoric/internal/src/lib-chainStorage.js'
+ * @import {MakeRecorderKit} from '@agoric/zoe/src/contractSupport/recorder.js'
+ * @import {USDCProposalShapes, ShareWorth} from '../pool-share-math.js'
+ */
+
+import { AmountMath } from '@agoric/ertp/src/amountMath.js';
+import {
+  AmountShape,
+  PaymentShape,
+  RatioShape,
+} from '@agoric/ertp/src/typeGuards.js';
```

Comment:
These can and should be imported from the top level

Reply:
yeah. I'm still suffering from bundle-size PTSD.

It doesn't make any difference in this case. 2.7M bundle size in either case. Something else is grabbing the rest of ERTP, or the difference is less than 1%.

If these imports are in a stand-alone entry point module, there is a noticeable difference (1.1M vs. 732K), but not as much as I expected.

I noticed that ERTP gets `M` and `matches` from `@agoric/store`. Changing that to `@endo/patterns` reduces it to 728K.